PowerEditor/src/ScitillaComponent/Buffer.cpp

```diff
@@ -1325,6 +1332,7 @@ LangType FileManager::detectLanguageFromTextBegining(const unsigned char *data,
 
 bool FileManager::loadFileData(Document doc, const TCHAR * filename, char* data, Utf8_16_Read * unicodeConvertor, LangType & language, int & encoding, EolType & eolFormat)
 {
+	NativeLangSpeaker *_nativeLangSpeaker = (NppParameters::getInstance())->getNativeLangSpeaker();

```

Comment:
Likewise, underscore prefix is used for class members not for method/block locals.